LocalPackages/AIChat/Sources/AIChat/TimerPixelHandler.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TimerPixelHandler.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+
+final class TimerPixelHandler {
```

Comment:
This is temporary. Will be removed once we ship the FE feature

Reply:
Is this something that should be put behind a subfeature flag and disabled once the FE support ships, or are we happy enough to just remove this code later?

Reply:
We're good in just removing the code later, once the new feature is released we'll also pair with a native release, so it should be good